tests/integration/support/client_wrapper.go

```diff
@@ -120,6 +122,12 @@ func GetInstance(apiOptions models.APIOptions) *ClientWrapper {
 	return clientInstance
 }
 
+// GetProjectConfig gets the right config project
+func (c *ClientWrapper) GetProjectConfig() config.ProjectConfig {
+	return c.projectConfig
```

Comment:
Instead of caching this, we can call `c.client.ConfigManager.GetConfig()`

Reply:
ok, addressed